main.go

```diff
@@ -44,12 +43,18 @@ const (
 var (
 	version  string
 	commitID string
+	debugBuild bool = true
```

Comment:
Let's convert this over to a `--verbose` flag in CLI arguments rather than as a build time constant.